Adds Gnollhack and adds Notdnethack as first ascension

nethack-cleaner opened the pull request:
Adds Gnollhack as variant and updates Notdnethacks list of ascension types and makes them first ascension listed.

The file schema/addgnoll.sql
will add the records for Gnollhack tracking.

Your max(logfiles_i) may be different than mine, in which case you would need to adjust that field prior to running it.

aoeixsz4 commented on nhdb-feeder.pl:
```diff
@@ -1453,6 +1453,11 @@ sub sql_player_name_map
     #--- parse log
 
       my $parsed_line = parse_log($log, $xlog_line);
+      if ($log->{'variant'} eq 'gnoll') {
```
what the heck is this part about?

aoeixsz4 commented on schema/addgnoll.sql:
```diff
@@ -0,0 +1,3 @@
+insert into logfiles (logfiles_i, descr, server, variant, logurl, localfile, dumpurl, oper, static, tz, fpos, lines)values (2025, 'gnollhack.org', 'hdf', 'gnoll', 'https://www.hardfought.org/xlogfiles/gnollhack/xlogfile', 'hdf.gnoll.log','https://www.hardfought.org/userdata/%X/%x/gnollhack/dumplog/%s.gnoll.txt', 't', 'f', 'UTC', 0, 0);
```
please add these to the already existing sources schema instead of adding a new flie

aoeixsz4 replied:
and just increase sequence number from the last double or triple-digit value

nethack-cleaner committed:
This adds Gnollhack as a variant and adds Notdnethack to the first ascension list and updates the playable combos fro notdnethack.

The file schema/addgnoll.sql
will add the records for Gnollhack tracking.

Your max(logfiles_i) may be different than mine, in which case you would need to adjust that field prior to running it.

nethack-cleaner commented:
Replacing with revision.
